cheatsheets_draft/Secrets_Management_CheatSheet.md

```diff
@@ -76,169 +76,180 @@ Secrets follow a lifecycle. The stages of the lifecycle are as follows:
+- Rotation/Temporality: credentials used by the CI/CD tooling to authenticate against the secret management system are rotated frequently and expire after a job completes.
+- Scope of authorization: scope credentials used by the CI/CD tooling (e.g. roles, users, etc.), e.g. only authorize those secrets and services of the secret management system required for the CI/CD tooling to execute its job.
+- Attribution of the caller: credentials used by the CI/CD tooling still hold attribution of the one calling the secrets management solution. Ensure you can attribute any calls made by the CI/CD tooling to a person or service that requested the actions of the CI/CD tooling. If this is not possible through the default configuration of the secrets manager, make sure that you have a correlation setup in terms of request parameters.
 - All of the above: Still follow those do's and don'ts listed in section 3.2.1: log & alert, take care of forking, etc.
+- Backup: back up secrets to product-critical operations in separate storage (f.e. cold storage), especially encryption keys.
 
 #### 3.2.3 Not touched by CI/CD at all
 
-Secrets do not necessarilty need to be brought to a consumer of the secret by a CI/CD pipeline. It is even better when the secret is actually retrieved by the consumer of the secret. In that case, the CI/CD pipeline still needs to instruct the orchestrating system (e.g. [Kubernetes](https://kubernetes.io/)) that it needs to schedule a certain service with a given service account with which the consumer can then retrieve the actual required secret. This means that the CI/CD tooling still has credentials towards the orchestrating platform, but no longer has access to the secrets themselves. The do's and don'ts regarding these type of credentials are similar to the ones described in section 3.2.2.
+Secrets do not necessarily need to be brought to a consumer of the secret by a CI/CD pipeline. It is even better when the consumer of the secret retrieves the secret. In that case, the CI/CD pipeline still needs to instruct the orchestrating system (e.g. [Kubernetes](https://kubernetes.io/)) that it needs to schedule a specific service with a given service account with which the consumer can then retrieve the required secret. The CI/CD tooling then still has credentials for the orchestrating platform but no longer has access to the secrets themselves. The do's and don'ts regarding these credentials types are similar to those described in section 3.2.2.
 
-### 3.3 Authentication and Authorization of CI/CD Tooling
+### 3.3 Authentication and Authorization of CI/CD tooling
```

Comment:
should this be a capital as part of the title? not sure... what would be a best practice here ?